### ISSUE TITLE
Error printed after command finishes when terminal-notifier is not installed

### ISSUE THREAD

**Issue by imrekoszo** [franciscolourenco/done#133]
M1 macos 13.0.1

When `terminal-notifier` is not installed, I get the following message printed after long-running commands. It went away when I installed the notifier.

```
142:145: syntax error: A identifier can’t go after this “"”. (-2740)
```

**Comment by franciscolourenco** [comment]
A fix will be released with v1.17.0

**Comment by imrekoszo** [comment]
Good man!
